wgpu-types/src/lib.rs

```diff
@@ -1472,29 +1475,35 @@ pub struct BindGroupEntry<R> {
 
 /// Describes a group of bindings and the resources to be bound.
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-pub struct BindGroupDescriptor<'a, L, B> {
+pub struct BindGroupDescriptor<'a, L, B>
+where
+    [B]: ToOwned<Owned = Vec<B>>,
```

Comment:
why is this needed? If we have `[B]` we should always be able to just call `to_vec()` and get a `Vec`, we shouldn't need these extra bounds

Reply:
I haven't investigated what the exact issue is yet, but if I leave the associated type constraint off, derives like `Debug` stop working